lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -885,3 +869,61 @@ describe('#mergeIamTemplates()', () => {
     });
   });
 });
+
+describe('#mergeIamTemplates() disableLogs flag', () => {
+  const withDisableLogsTrueFunctionName = 'withDisableLogsTrue';
+  const withDisableLogsFalseFunctionName = 'withDisableLogsFalse';
+  const withoutDisableLogsFunctionName = 'withoutDisableLogs';
+  let serverless = null;
+  let withDisableLogsFalseFunction = null;
+  let withoutDisableLogsFunction = null;
+  let compiledCloudFormationTemplate = null;
+  before(() => {
+    return runServerless({
+      cwd: 'tests/fixtures/functionDisableLogs',
```

Comment:
Let's rely on dedicated `fixtures` handler, also instead of creating a new dedicated fixture for this case, let's use simple [`function` fixture](https://github.com/serverless/serverless/tree/d43af5d3ea78bc37eee363f8da5db62c705205d7/tests/fixtures/function) I've just added, and extend it, as it's done here: https://github.com/serverless/serverless/blob/4c2a52d1bf8fdb15683c09a8db800aa0e5842950/lib/plugins/aws/package/compile/events/httpApi/index.test.js#L119

That way we will not pollute _fixtures_ folder with tons of fixtures that serve small individual cases

Reply:
Aaaah yes, I noticed it when I was working on another PR
Will re-push